src/main/java/com/twitter/http2/DefaultHttp2Headers.java

```diff
@@ -0,0 +1,65 @@
+package com.twitter.http2;
+
+import io.netty.handler.codec.DefaultHeaders.NameValidator;
+import io.netty.handler.codec.http.DefaultHttpHeaders;
+import io.netty.util.AsciiString;
+import io.netty.util.ByteProcessor;
+import io.netty.util.internal.PlatformDependent;
+
+import static io.netty.util.internal.ObjectUtil.checkNotNull;
+
+public class DefaultHttp2Headers extends DefaultHttpHeaders {
+  private static final ByteProcessor HEADER_NAME_VALIDATOR = new ByteProcessor() {
+    @Override
```

Comment:
Ah ok I missed the ":" so this is different then from the DefaultHttp2Headers in the main netty project where they don't check all of this? Do we know why?